Mobile Buy SDK Sample Apps/Sample App Advanced/Mobile Buy SDK Advanced Sample/ShippingRatesTableViewController.m

```diff
@@ -64,27 +62,47 @@ - (void)viewDidLoad {
     
     [self.tableView registerClass:[ShippingRateTableViewCell class] forCellReuseIdentifier:@"Cell"];
     
-    // Setup both operations to run
-    GetShopOperation *shopOperation = [[GetShopOperation alloc] initWithClient:self.client];
-    shopOperation.delegate = self;
-    [[NSOperationQueue mainQueue] addOperation:shopOperation];
-    
-    GetShippingRatesOperation *shippingOperation = [[GetShippingRatesOperation alloc] initWithClient:self.client withCheckout:self.checkout];
-    shippingOperation.delegate = self;
-    [[NSOperationQueue mainQueue] addOperation:shippingOperation];
-    
     [[UIApplication sharedApplication] setNetworkActivityIndicatorVisible:YES];
-    
-    // Ensure both operations are completed before we reload the table view
-    NSBlockOperation *blockOperation = [NSBlockOperation blockOperationWithBlock:^{
+    [self getShopAndShippingRates];
+}
+
+- (void)getShopAndShippingRates
+{
+    [self.client getShop:^(BUYShop *shop, NSError *error) {
+        if (shop && !error) {
+            [self didGetShop:shop];
+            [self getShippingRates];
```

Comment:
I'm wondering if we can use `dispatch_group_t` instead to fetch the shop and shipping rates in parallel instead of sequencing them.